net-misc/turbovnc/files/turbovnc-2.2.7-fix-musl-compilation.patch

```diff
@@ -0,0 +1,36 @@
+See bug https://bugs.gentoo.org/836723
+
+There are 2 compilation errors:
+
+/var/tmp/portage/net-misc/turbovnc-2.2.7-r2/work/turbovnc-2.2.7/unix/Xvnc/programs/Xserver/os/access.c: In function 'GetLocalClientCreds':
+/var/tmp/portage/net-misc/turbovnc-2.2.7-r2/work/turbovnc-2.2.7/unix/Xvnc/programs/Xserver/os/access.c:1178:18: error: storage size of 'peercred' isn't known
+ 1178 |     struct ucred peercred;
+      |                  ^~~~~~~~
+
+/var/tmp/portage/net-misc/turbovnc-2.2.7-r2/work/turbovnc-2.2.7/unix/Xvnc/programs/Xserver/Xext/xf86bigfont.c:48:10: fatal error: asm/page.h: No such file or directory
+   48 | #include <asm/page.h>
+      |          ^~~~~~~~~~~~
+
+
+--- a/unix/Xvnc/programs/Xserver/os/access.c
++++ b/unix/Xvnc/programs/Xserver/os/access.c
+@@ -84,6 +84,8 @@
+ #include <X11/Xwinsock.h>
+ #endif
+ 
++#define _GNU_SOURCE
```

Comment:
We should set this in the build system ideally if it's needed, but I guess this is actually from Xserver somewhere?

Reply:
My first attempt was to add `-DCMAKE_C_FLAGS="-D_GNU_SOURCE"` to `mycmakeargs` but got the QA error about not respecting CFLAGS, so I made a patch instead.

Reply:
I patched the build system.